compiler/fir/analysis-tests/testData/resolve/overrides/simple.kt

```diff
@@ -8,7 +8,7 @@ open class A {
 
 class B : A() {
     override fun foo(): B = this
-    fun bar(): B = this // Ambiguity, no override here (really it's just "missing override" and no ambiguity)
+    fun <!VIRTUAL_MEMBER_HIDDEN!>bar<!>(): B = this // Ambiguity, no override here (really it's just "missing override" and no ambiguity)
```

Comment:
The comment here seems outdated if there is no ambiguity (maybe there was before)

Reply:
Thanks! Yeah this is confusing. Let me change it to simply `Missing 'override'`.